README.md

```diff
@@ -0,0 +1,80 @@
+# asset-depreciation-calculator
+
+> Calculate the asset depreciation
+
+## Installation
+
+```console
+$ npm install asset-depreciation-calculator
+```
+
+## Usage
+
+```js
+const calculateDepreciation = require("asset-depreciation-calculator");
+
+calculateDepreciation(
+  20000,
+  "2016-08-01",
+  5,
```

Comment:
🔧  I think it would be more readable if we use an object as an argument, otherwise it's unclear what every argument means:
```javascript
calculateDepreciation({
  purchaseAmount: 20000,
  purchaseDate: "2016-08-01",
  years: 5
})
```

Reply:
thanks, it's updated https://github.com/kontist/asset-depreciation-calculator/pull/1/commits/62bce127c60a700f27aa15eb2362d1a1549ff1d3